### PR TITLE
bug 1590486 volume security for Cinder

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1590486

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_cinder.adoc [hunk 1]
```diff
@@ -87,55 +81,50 @@ spec:
...
 Users can then xref:../../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[request storage
 using persistent volume claims], which can now utilize your new persistent
 volume.
 
 [IMPORTANT]
 ====
-Persistent volume claims only exist in the user's namespace and can only be
-referenced by a pod within that same namespace. Any attempt to access a
+Persistent volume claims exist only in the user's namespace and can be
+referenced by only a pod within that same namespace. Any attempt to access a
```
Comment (qinpingli) [inline install_config/persistent_storage/persistent_storage_cinder.adoc]: s/by only a pod/by a pod/
PVC can be used by multi pods.

File: install_config/persistent_storage/persistent_storage_cinder.adoc [hunk 2]
```diff
@@ -144,3 +133,61 @@ formatted with the given file system.
...
+          protocol: TCP
+      restartPolicy: Always
+      serviceAccountName: <service_account> <1>
+      securityContext:
+        fsGroup: 7777 <2>
+----
+<1> Specify the service account you created.
+<2> Specify an
+xref:../../install_config/persistent_storage/pod_security_context.adoc#fsgroup[`*fsGroup*`]
+for the pods.
```
Comment (qinpingli) [inline install_config/persistent_storage/persistent_storage_cinder.adoc]: There are 2 labels of <1> and <2>, and no explain for <3>, <4>, <5>

**Comment by kalexand-rh** [comment]
Thanks @qinpingli! I've updated the topic.
@openshift/team-documentation, PTAL

**Comment by ahardin-rh** [comment]
LGTM! :fireworks:

**Comment by kalexand-rh** [comment]
The logic of this change was verified on the source bug. Thank you both! Merging. :)

**Comment by kalexand-rh** [comment]
3.10: https://github.com/openshift/openshift-docs/pull/10559
3.9: https://github.com/openshift/openshift-docs/pull/10560
3.7: https://github.com/openshift/openshift-docs/pull/10561

**Pull request merged**